clients/redshift/merge.go

```diff
@@ -84,12 +84,10 @@ func (s *Store) Merge(ctx context.Context, tableData *optimization.TableData) er
 
 		err = utils.BackfillColumn(ctx, s, col, tableData.ToFqName(ctx, s.Label(), true))
 		if err != nil {
-			defaultVal, _ := col.DefaultValue(ctx, nil)
-			return fmt.Errorf("failed to backfill col: %v, default value: %v, error: %v",
-				col.Name(ctx, nil), defaultVal, err)
+			return fmt.Errorf("failed to backfill col: %v, default value: %v, error: %v", col.RawName(), col.RawDefaultValue(), err)
```

Comment:
`%w` for error?

Reply:
Let's keep it the same for now? There's not much benefit from changing this yet

Reply:
Also `err` instead of `error`, I think that's our convention?

Reply:
True will fix